.github/workflows/vscode-extension-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: "VSCode Extension - CI"
+
+on:
+  push:
+    branches:
+      - task/extension/branch-preview
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: 18.x
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.x
+
+    - name: Install dependencies
+      run: npm i -g typescript @vscode/vsce
+
+    - name: Run PowerShell script
+      run: |
+        pwsh -File ./scripts/execute-vscode-sideload.ps1
```

Comment:
this is going to sideload into the GitHub actions runner, I'm not sure how it helps us?

Reply:
I am following the instructions inside the debugging.md file.
I want to see whether sideloading would sideload into a codespaces for the branch.
I haven't set up the codespaces here yet. I am currently making sure that the prerequisites are in the worflow

Reply:
I think to achieve that you'd need to run the sideloading script inside of the container definition.

Reply:
Yes... Learning that too right now...